qsdsan/sanunits/_clarifier.py

```diff
@@ -1040,12 +1040,13 @@ def _f_i(self):
         fx = xcod/self._mixed.COD
         corr = self._corr
         HRT = self._HRT
-        n_COD = corr*(2.88*fx - 0.118)*(1.45 + 6.15*np.log(HRT*24*60))
+        # n_COD should not be negative
+        n_COD = max(0, corr*(2.88*fx - 0.118)*(1.45 + 6.15*np.log(HRT*24*60)))
```

Comment:
@RaiSaumitra I added this line because I found out `n_COD` could be negative for some flows. Is it from small rounding errors or because there's something wrong with the flows?

Reply:
When `fx` < 0.041, `n_COD` becomes negative. So I think we should differentiate these two cases:

1. `fx` < 0: something's wrong with the `_mixed` stream
2. 0 <=`fx` <0.041: solids are too dilute. n_COD can be set to 0, i.e., PC does not effectively remove particulate COD. I think we can allow this to happen, but it creates a tricky situation for underflow composition (i.e., contains no solids). The underflow should at least have the same solid concentrations as `_mixed`, if not more. So, I suggest we do a sanity check in `_run` to ensure/enforce `1-f_i >= r` or `f_i <= 1-r`.

Reply:
@joyxyz1994 This sounds good to me, although I don't quite follow why `1-f_i >= r` is equivalent to `0 <=fx <0.041`?